e2e/wallets.e2e-spec.ts

```diff
@@ -52,6 +56,10 @@ describe('Wallets', () => {
     expect<any>(page.loadWalletCheckValidationLabel()).toEqual(false);
   });
 
+  it('should load wallet with already used seed', () => {
```

Comment:
The "not" is missing, it should say "should not load..."

Reply:
Fixed.